test/models/orchestration/dhcp_test.rb

```diff
@@ -75,7 +75,7 @@ def teardown
   test "provision interface DHCP records should contain default filename/next-server attributes for IPv4 tftp proxy" do
     ProxyAPI::TFTP.any_instance.expects(:bootServer).returns('192.168.1.1')
     subnet = FactoryGirl.build(:subnet_ipv4, :dhcp, :tftp)
-    h = FactoryGirl.create(:host, :with_dhcp_orchestration, :with_tftp_dual_stack_orchestration, :subnet => subnet)
+    h = FactoryGirl.create(:host, :with_dhcp_orchestration, :with_tftp_orchestration, :subnet => subnet)
```

Comment:
@timogoebel could you have a look here?

Reply:
@mmoll: Does this break anything? Basically, the change is fine. The behaviour should be well enough covered in the TFTP tests. This just tests, that the behaviour doesn't change if TFTP orchestration is enabled on the IPv6 subnet.

Reply:
without this change:
`ActiveRecord::SubclassNotFound: Invalid single-table inheritance type: Subnet::Ipv4 is not a subclass of Subnet::Ipv6`

Reply:
Ouch, I have a feeling that we should look into that rather than changing the test.

We do some pretty nasty stuff for the subnet classes:
https://github.com/theforeman/foreman/blob/develop/app/models/concerns/foreman/sti.rb &
https://github.com/theforeman/foreman/blob/develop/app/models/subnet.rb#L281-L287

Reply:
I did add `as_admin` to the test and it's working now